### PR TITLE
feat: widgets faq

### PR DESCRIPTION
**Pull request by solidovic**
## Description

New collection structure and preview for FAQ widgets.

See also warehouse: https://github.com/lidofinance/warehouse/pull/161

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 # Local development (files are saved locally)
 
 - Install dependencies with `yarn`.
+- Get `lido-landing` directory from [lido-content repo](https://github.com/lidofinance/lido-content) to root.
```
Comment (kozlovzxc-at-lido) [inline README.md:6]: I guess this content is the same as https://github.com/lidofinance/lido-cms/pull/24

**Review by kozlovzxc-at-lido — DISMISSED** [review]
Looks ok for me, but i'm not really familiar with codebase, please wait for @Slavik191 review

File: README.md (hunk 1 above)
Comment (solidovic) [reply README.md:6]: You are right

File: features/cms/previews/faqWidgets/faqWidgets.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import { Wrapper, Text, Bold, Box } from "./faqWidgetsStyles";
+
+type Pages = {
+  identification?: string;
+  faq?: {
+    question: string;
+    answer: string;
+  }[];
+}[];
```
Comment (Slavik191) [inline features/cms/previews/faqWidgets/faqWidgets.tsx:15]:
```suggestion
type Data = {
  Pages: {
    identification?: string;
    faq?: {
      question: string;
      answer: string;
    }[];
  }[];
}
```

File: features/cms/previews/faqWidgets/faqWidgets.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+export const FaqWidgetsPreview = ({ entry }: PreviewTemplateComponentProps) => {
+  const pages: Pages = entry.getIn(["data", "pages"]).toJS();
```
Comment (Slavik191) [inline features/cms/previews/faqWidgets/faqWidgets.tsx:18]:
```suggestion
  const { pages } : Data = entry.getIn(["data"]).toJS();
```

File: features/cms/previews/faqWidgets/faqWidgets.tsx (hunk 2 above)
Comment (solidovic) [reply features/cms/previews/faqWidgets/faqWidgets.tsx]: done

File: features/cms/previews/faqWidgets/faqWidgets.tsx (hunk 3 above)
Comment (solidovic) [reply features/cms/previews/faqWidgets/faqWidgets.tsx]: done

File: README.md [hunk 4]
```diff
@@ -3,6 +3,9 @@
...
+  - You should make it 'cause a current structure made for [lido-content repo](https://github.com/lidofinance/lido-content).
+  - Or you should change structure in [features/cms/initialize](features/cms/initialize) and [features/cms/previews](features/cms/previews)
```
Comment (Slavik191) [inline README.md:8]:
```suggestion
- To add content, copy the folders (of the desired projects) from the [content repository](https://github.com/lidofinance/lido-content) to the root. (optional)
```

**Review by Slavik191 — APPROVED** [review]

**Pull request merged**